### DISCUSSION TITLE
Flatpak request

### DISCUSSION THREAD

**Discussion by mcexit** [ananthakumaran/paisa#207] [Ideas]
This is a request to create a flatpak of the "desktop" app. If you have time to work on this, maybe a self-contained flatpak initially with  your releases would be ideal.

The docs for Flatpak app authors can be found here:

https://docs.flathub.org/docs/category/for-app-authors

Once you're ready, having your app available on the Flathub repo would be awesome and garner some user adoption and possible contributors.

> Side note: Personally there are things I don't like about Flatpak, but your app seems like the ideal candidate to package as one.

**Comment by airtonix** [comment]
Since this is a golang project, this can just be easily automated with goreleaser. 

it's insanely easy to do and it'll manage building and uploading allllll kinds of linux/mac/windows bundles to the release page.
